### ISSUE TITLE
Support filters on nested queries

### ISSUE THREAD

**Issue by asselstine** [graphprotocol/graph-node#1304]
**Do you want to request a *feature* or report a *bug*?**

Feature

**What is the current behavior?**

Filters are only supported at the top level

**What is the expected behavior?**

Filters should be supported for every collection:

```graphql
query drawsQuery {
  players{
    id
    entries(where:{ drawId_lte: 3}){
      drawId
      balance
    }
  }  
}
```

**Comment by leoyvens** [comment]
This was actually fixed in #1212, but that hasn't made into our production service yet, which I assume is where you reproduced this. It should get released this week.

**Comment by asselstine** [comment]
@leoyvens Fantastic!
